.devcontainer/devcontainer.json

```diff
@@ -54,10 +54,10 @@
 
 	// Use 'postCreateCommand' to run commands after the container is created.
 	// Installs:
-	//   1. Mega Linter
+	//   1. Mega Linter`
 	//   2. Pester
 	//   3. Configures benchpress Python module
-	"postCreateCommand": "npm install -g mega-linter-runner && pwsh -command Install-Module -Name Pester -Force -SkipPublisherCheck && pip install --editable ./framework/python/",
+	"postCreateCommand": "npm install -g mega-linter-runner && pwsh -command Install-Module -Name Pester -Force -SkipPublisherCheck && pip install --editable ./framework/python/ && Install-Module Az",
```

Comment:
The [docs](https://code.visualstudio.com/docs/devcontainers/create-dev-container#_rebuild) state that its more efficient for us to be adding these installes into the Dockerfile and not in the `postCreateCommand`, would you be able to make that change here?

Reply:
one or two post create command(s) is cool, 4 is too much

Reply:
I will touch base with Uffaz and do necessary changes

Reply:
After removing PowerShell commands into Dockerfile, docker image builds successfully but run commands in the docker file does not get executed after the image is built. Currently investigating it

Reply:
Timebox investigation here to a couple hours, feel free to file an issue to fix this later, not required for merging this PR